jsapp/js/components/submissions/SubmissionBackgroundAudio.tsx

```diff
@@ -0,0 +1,63 @@
+import AttachmentActionsDropdown from '#/attachments/AttachmentActionsDropdown'
+import DeletedAttachment from '#/attachments/deletedAttachment.component'
+import bem from '#/bem'
+import AudioPlayer from '#/components/common/audioPlayer'
+
+import { hasBackgroundAudioEnabled } from '#/assetUtils'
+import type { AssetResponse, SubmissionAttachment } from '#/dataInterface'
+import type { SubmissionResponse, SurveyRow } from '#/dataInterface'
+
+interface SubmissionBackgroundAudioProps {
+  asset: AssetResponse
+  submission: SubmissionResponse
+  survey: SurveyRow[]
```

Comment:
I think if you have `asset` here, you don't need `survey`, as you can use `asset.content.survey` (with checks) :)